padle/padle.py

```diff
@@ -0,0 +1,537 @@
+        if ctx.author.id not in subbed_users:
+            async with self.config.subs() as subs:
+                subs.append(ctx.author.id)
+            return await send_confirmation_message(ctx, "You will now recieve notifications of new PADles. "
+                                                   "You can unsubscribe with `{}padle subscribe 0`.".format(prefix))
+
+        confirmation = await get_user_confirmation(ctx, "You are already subscribed. Did you mean to unsubscribe?")
+        if confirmation:
+            await self.unsubscribe(ctx)
+        elif confirmation is None:
+            await send_cancellation_message(ctx, "Confirmation timeout")
+        else:
+            await ctx.send("No changes were made, you will still recieve notifications of new PADles.")
+
+    async def unsubscribe(self, ctx):
+        async with self.config.subs() as subs:
+            subs.remove(ctx.author.id)
+        return await send_confirmation_message(ctx, "You will no longer recieve notifications of new PADles.")
+
+    @padle.command()
+    async def start(self, ctx):
+        """Start a game of PADle"""
+        if(await self.config.user(ctx.author).done()):
+            return await ctx.send("You have already finished today's PADle!")
+        if(await self.config.user(ctx.author).start()):
+            return await ctx.send("You have already started a game!")
+        confirmation = await get_user_confirmation(ctx, "Start today's (#{}) PADle game?".format(
+            await self.config.num_days()), timeout=30, force_delete=False, show_feedback=True)
+        if confirmation is None:
+            return await send_cancellation_message(ctx, "Confirmation timeout.")
+        if await self.config.user(ctx.author).start():
+            return await send_cancellation_message(ctx, "You cannot start multiple games!")
+        if not confirmation:
+            return
+
+        prefix = (await self.bot.get_prefix(ctx.message))[0]
+        em = discord.Embed(title="PADle #{}".format(await self.config.num_days()), type="rich",
+                           description="Guess a card with `{}padle guess <card>`!\nIf "
+                           "you give up, use `{}padle giveup`.".format(prefix,prefix))
+        try:
+            message = await ctx.author.send(embed=em)
+        except discord.Forbidden:
+            return await send_cancellation_message(ctx, "Looks like I can't DM you. Try checking your Privacy Settings.")
+        if ctx.guild is not None:
+            await send_confirmation_message(ctx, "Check your DMs!")
+        await self.config.user(ctx.author).start.set(True)
+        await self.config.user(ctx.author).edit_id.set(message.id)
+        await self.config.user(ctx.author).channel_id.set(message.channel.id)
+        await self.config.user(ctx.author).embed_text.set([])
+
+    @padle.command(aliases=["stats"])
+    async def globalstats(self, ctx, day=0):
```

Comment:
The same goes for the integer here.  The argument will be interpreted as a string unless you explicitly tell it to be an integer via `day: int = 0`